docs/howtos/layers/tracks.md

```diff
@@ -121,66 +121,6 @@ napari.run()
 * Graph - check this box to display a previously created graph as explained in
   [](#arguments-of-view_tracks-and-add_tracks).
 
-## Arguments of `view_tracks` and `add_tracks`
```

Comment:
These docs are incomplete and stale for multiple reasons (including properties/features), so I removed this in favor of link to the two functions in the text above.

Reply:
lmao I came here while trying to resolve merge conflicts with #420 going, "why on earth did this get removed. And no discussion either, wth!"

... Then I found my thumbs-up above 🤣